minigalaxy/ui/gametile.py

```diff
@@ -272,10 +329,18 @@ def __get_install_dir(self):
 
     def reload_state(self):
         self.game.install_dir = self.__get_install_dir()
-        dont_act_in_states = [self.state.QUEUED, self.state.DOWNLOADING, self.state.INSTALLING, self.state.UNINSTALLING]
+        dont_act_in_states = [self.state.QUEUED, self.state.DOWNLOADING, self.state.INSTALLING, self.state.UNINSTALLING,
+                              self.state.UPDATING, self.state.DOWNLOADING]
         if self.current_state in dont_act_in_states:
             return
-        if self.game.install_dir and os.path.exists(self.game.install_dir):
+        if self.game.installed_version:
```

Comment:
When ``self.game.id`` is set to 0, this shouldn't be executed. I'm not sure if it would execute again after the id is loaded from the API, though.

Reply:
This happens with Pixeljunk Shooter. The gameinfo file for that game looks like this:
```
PixelJunk Shooter
2.0.0.3
r971
```